src/NSwag.ConsoleCore/NSwag.ConsoleCore.csproj

```diff
@@ -16,12 +16,23 @@
     <PackageId>NSwag.ConsoleCore</PackageId>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\NSwag.AspNetCore.Launcher\NSwag.AspNetCore.Launcher.csproj" />
+    <ProjectReference Include="..\NSwag.AspNetCore.Launcher\NSwag.AspNetCore.Launcher.csproj" PrivateAssets="All" />
     <ProjectReference Include="..\NSwag.Commands\NSwag.Commands.csproj" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NConsole" Version="3.9.6519.30868" />
     <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
     <PackageReference Include="System.Net.Http" Version="4.3.2" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  <!-- Workaround for https://github.com/NuGet/Home/issues/3891 -->
+  <PropertyGroup>
```

Comment:
Didn't realize this produced a nupkg that was uploaded. I'm adding the outputs of the launcher to this package so you can execute the AspNetCoreToSwaggerCommand. Ideally, we'd want this to be part of the NSwag.Commands project, but this changes none of the build order.